__tests__/help.ts

```diff
@@ -1,11 +1,16 @@
-import { exec } from 'child_process';
+import util from 'util';
 
-test('npx unimported --help', (done) => {
+const exec = util.promisify(require('child_process').exec);
+
+test('npx unimported --help', async (done) => {
   // note about `./` path: jest executes the tests from the root directory
-  exec('node ./bin/unimported.js --help', (error, stdout, stderr) => {
-    expect(error).toBe(null);
-    expect(stderr).toBe('');
-    expect(stdout.trim()).toMatchInlineSnapshot(`
+  const { stdout, stderr, error } = await exec(
+    'node ./bin/unimported.js --help',
+  );
+
+  expect(error).toBe(undefined);
+  expect(stderr).toBe('');
```

Comment:
`error` is always going to be `undefined` now as the promise will reject if an error is returned and this line won't be hit.  You can just remove this assertion now.